### PR TITLE
Get tot num params from storage

### PR DESCRIPTION
**Pull request by dafeda**
## Pre review checklist

- [ ] Read through the code changes carefully after finishing work
- [ ] Make sure tests pass locally (after every commit!)
- [ ] Prepare changes in small commits for more convenient review (optional)
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: src/ert/config/parameter_config.py [hunk 1]
```diff
@@ -40,6 +40,11 @@ def sample_or_load(
     ) -> xr.Dataset:
         return self.read_from_runpath(Path(), real_nr)
 
+    @property
+    @abstractmethod
+    def size(self) -> int:
```
Comment (oyvindeide) [inline src/ert/config/parameter_config.py:45]: Could be `__len__` perhaps?

**Review by oyvindeide — APPROVED** [review]
LGTM! As we discussed this is valid as long as the parameters are used to drive the model, and the only public facing way of doing this is though: https://github.com/equinor/ert/blob/4aaa3124f64ec0bf59e4c5bc46ca4e22e2086f30/src/ert/shared/hook_implementations/workflows/disable_parameters.py#L11, it would be good to add a sentence there just to document it.

File: src/ert/config/parameter_config.py (hunk 1 above)
Comment (dafeda) [reply src/ert/config/parameter_config.py]: Sure thing.

**Comment by dafeda** [comment]
> LGTM! As we discussed this is valid as long as the parameters are used to drive the model, and the only public facing way of doing this is though:
> 
> https://github.com/equinor/ert/blob/4aaa3124f64ec0bf59e4c5bc46ca4e22e2086f30/src/ert/shared/hook_implementations/workflows/disable_parameters.py#L11
> 
> , it would be good to add a sentence there just to document it.

Where would you document this?
`disable_parameters.py` states that it will remove parameters from the udpate and does not deal with "driving" the model, so it feels wrong to put this information there.

**Comment by dafeda** [comment]
> > Where would you document this?
> > `disable_parameters.py` states that it will remove parameters from the udpate and does not deal with "driving" the model, so it feels wrong to put this information there.
> 
> I was thinking something along the lines of:
> 
> > Disabling parameters means they are not updated, however it is assumed that they impact the model, so they are included if projection is calculated.
> 
> But that might cause more confusion than actual help, so maybe we should drop it.

I think that may cause more confusion as we would have to define projection..

**Comment by oyvindeide** [comment]
> I think that may cause more confusion as we would have to define projection..

Agreed, leave it out 👍

**Pull request merged**